potassco/reify.h

```diff
@@ -0,0 +1,150 @@
+// {{{ MIT License
+
+// Copyright 2017 Roland Kaminski
```

Comment:
The preambles should probably be aligned to what Benjamin is using. They should be added to all files. Maybe remove the year. Anyone who changes the files should add his name.